frontend/app/component/form/polling_station/PollingStationForm.tsx

```diff
@@ -0,0 +1,150 @@
+            <input type="hidden" id="id" name="id" defaultValue={pollingStation?.id} />
+
+            {/* props that are not in design but are in the pollingstation model */}
+            <div className="hidden">
+              <input type="text" id="house_number" name="house_number" defaultValue={pollingStation?.house_number} />
+            </div>
+
+            <FormLayout.Row>
+              <InputField
+                id="number"
+                type="number"
+                name="number"
+                label={t("number")}
+                fieldWidth="narrow"
+                defaultValue={pollingStation?.number}
+              />
+              <InputField id="name" name="name" label={t("name")} defaultValue={pollingStation?.name} />
+            </FormLayout.Row>
+
+            <FormLayout.Field>
+              <ChoiceList>
+                <ChoiceList.Title>{t("polling_station.title.type")}</ChoiceList.Title>
+                {labelForPollingStationType.entries.map((entry) => (
+                  <ChoiceList.Radio
+                    key={entry.key}
+                    id={`polling_station_type-${entry.key}`}
+                    name={"polling_station_type"}
+                    defaultValue={entry.key}
+                    defaultChecked={pollingStation?.polling_station_type === entry.key}
+                    label={entry.value}
+                  />
+                ))}
+              </ChoiceList>
+            </FormLayout.Field>
+
+            <InputField
+              id="number_of_voters"
+              name="number_of_voters"
+              label={t("election_status.number_of_voters")}
+              subtext={t("optional")}
+              fieldWidth="narrow-field"
+              defaultValue={pollingStation?.number_of_voters}
+              numberInput
+            />
+          </FormLayout.Section>
+
+          <FormLayout.Section title={t("polling_station.title.address")}>
+            <InputField
+              id="street"
+              name="street"
+              label={t("polling_station.street_and_number")}
+              defaultValue={pollingStation?.street}
+            />
+            <FormLayout.Row>
+              <InputField
+                id="postal_code"
+                name="postal_code"
+                fieldWidth="narrow"
+                label={t("polling_station.zipcode")}
+                defaultValue={pollingStation?.postal_code}
+              />
+              <InputField
+                id="locality"
+                name="locality"
+                label={t("polling_station.locality")}
+                defaultValue={pollingStation?.locality}
+              />
+            </FormLayout.Row>
+          </FormLayout.Section>
+
+          <FormLayout.Controls>
+            <Button type="submit" name="submit">
+              {pollingStation ? t("polling_station.form.save_update") : t("polling_station.form.save_create")}
+            </Button>
```

Comment:
I'm missing a cancel button for when I update a polling station and change my mind.

Reply:
I had discussed this with @jorisleker, I think the conclusion then was that the browser back button works.

When working on this I do miss it, specifically when you open de page via a direct (deep) link.